src/hangar/dataloaders/common.py

```diff
@@ -113,14 +113,14 @@ def get_shapes(self, converter=None):
 
         Returns
         -------
-        A tuple of arrayset shapes
+        A tuple of column shapes
         """
         shapes = []
-        for aset in self.arrayset_array:
-            if aset.variable_shape:
-                aset_shape = (None,) * len(aset.shape)
+        for col in self.columns_col:
+            if col.column_layout == 'variable_shape':
```

Comment:
This should be `col.schema_type`

Reply:
Ugh. Thanks!

Such a stupid mistake to make... I spent way more time trying to get that test to work then you would like to know...